Gopkg.template.toml

```diff
@@ -0,0 +1,28 @@
+[[constraint]]
```

Comment:
Are these Gopkg changes needed as part of this change?

Reply:
No, I plan to remove them before marging, although I definitely recommend trying to eliminate the use of our private Terraform fork as a separate change.